### PR TITLE
support invalidate all queries

### PR DESCRIPTION
**Pull request by datner**
Closes: an issue that bothered me

### What are the changes and their implications?

Support `invalidateQueries` not requiring any input, mirroring the original api
before
```ts
invalidateQueries(resolver, params)
invalidateQueries(resolver)
getQueryClient().invalidateQueries()
```
after
```ts
invalidateQueries(resolver, params)
invalidateQueries(resolver)
invalidateQueries()
```

## Bug Checklist

- [x] Changeset added (run `pnpm changeset` in the root directory)
- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [ ] Changeset added (run `pnpm changeset` in the root directory)
- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [ ] Documentation added/updated (submit PR to [blitzjs.com repo `main` branch](https://github.com/blitz-js/blitzjs.com))

### CODE REVIEW

**Comment by datner** [comment]
@beerose any updates?

**Comment by dillonraphael** [comment]
Hey @datner pretty straight forward PR. Just need to fix the pnpmlock issue with the CI

File: .changeset/stupid-rabbits-jump.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@blitzjs/rpc": minor
```
Comment (dillonraphael) [inline .changeset/stupid-rabbits-jump.md:2]:
```suggestion
"@blitzjs/rpc": patch
```

**Review by dillonraphael — CHANGES_REQUESTED** [review]

File: .changeset/stupid-rabbits-jump.md (hunk 1 above)
Comment (datner) [reply .changeset/stupid-rabbits-jump.md]: committed 🤓

**Comment by datner** [comment]
@dillonraphael no idea how to solve the pnplock issue, had it for all my prs..
I would greatly appreciate it if you could do it for me or guide me through it
Feel free to commit in my pr btw, I don't mind

**Review by dillonraphael — APPROVED** [review]

**Pull request merged**